src/Symfony/Component/Console/Style/StyleInterface.php

```diff
@@ -91,7 +91,7 @@ public function table(array $headers, array $rows);
      * @param string|null   $default
      * @param callable|null $validator
      *
-     * @return string
+     * @return mixed
```

Comment:
Can it really be anything (=`mixed`) like `resource` or `SplFileObject` ?

Reply:
"mixed" is technically correct, because of https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Question/Question.php#L58
I'm a bit sad that this will hide the intend - because only strings are returned as answers usually...